### ISSUE TITLE
C:\Windows\system32\wwwroot\

### ISSUE THREAD

**Issue by SimplyAName** [RatScanner/RatScanner#153]
**Error**
C:\Windows\system32\wwwroot\
```
System.IO.DirectoryNotFoundException: C:\Windows\system32\wwwroot\
   at Microsoft.Extensions.FileProviders.PhysicalFileProvider..ctor(String root, ExclusionFilters filters)
   at Microsoft.AspNetCore.Components.WebView.Wpf.BlazorWebView.StartWebViewCoreIfPossible()
   at Microsoft.AspNetCore.Components.WebView.Wpf.BlazorWebView.OnApplyTemplate()
   at System.Windows.FrameworkElement.ApplyTemplate()
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at System.Windows.Controls.Grid.MeasureOverride(Size constraint)
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at MS.Internal.Helper.MeasureElementWithSingleChild(UIElement element, Size constraint)
   at System.Windows.Controls.ContentPresenter.MeasureOverride(Size constraint)
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Wind
```
<details>
<summary>Log</summary>

```
[18:35:46.1463452] > [Info]  ----- RatScanner 3.0.0 -----
[18:35:46.1465792] > [Info]  Starting RatScanner...
[18:35:46.1465852] > [Info]  Prewarming name tool tip...
[18:35:46.1625427] > [Info]  Prewarming icon tool tip...
[18:35:46.1798548] > [Info]  Checking for new updates...
[18:35:46.2173806] > [Info]  Loading resource "RSClientVersion"...
[18:35:46.9797103] > [Info]  Loading config...
[18:35:46.9878698] > [Info]  Loading item data...
[18:35:46.9880052] > [Info]  Loading resource "RSItemDataVersion"...
[18:35:47.0902959] > [Info]  A new item data version is available: 20210630
[18:35:47.0903057] > [Info]  Loading resource "RSItemDataLink"...
[18:35:47.1831410] > [Info]  Downloading file "https://api.ratscanner.com/v3/file/items-en-30.06.2021.json"...
[18:35:50.4997964] > [Info]  Loading TarkovTracker...
[18:35:52.3969844] > [Info]  Loading progress data...
[18:35:52.7534541] > [Info]  Loading price data...
[18:35:53.3037051] > [Info]  Setting default item...
[18:35:53.3039850] > [Info]  Initializing RatEye...
[18:35:56.5587340] > [Info]  Initializing hotkey manager...
[18:35:56.5607313] > [Info]  Setting up data update routines...
[18:35:56.5607438] > [Info]  Ready!
[18:36:30.0661409] > [Info]  Saving config...
[18:36:30.0793506] > [Info]  Config saved!
[18:36:50.5680925] > [Info]  Saving config...
[18:36:50.5807949] > [Info]  Config saved!
[18:37:12.0156428] > [Info]  Saving config...
[18:37:12.0263760] > [Info]  Config saved!
[18:37:27.5039690] > [Info]  Saving config...
[18:37:27.5141238] > [Info]  Config saved!
[18:46:37.0611259] > [Error] C:\Windows\system32\wwwroot\
 -------------------- 
 System.IO.DirectoryNotFoundException: C:\Windows\system32\wwwroot\
   at Microsoft.Extensions.FileProviders.PhysicalFileProvider..ctor(String root, ExclusionFilters filters)
   at Microsoft.AspNetCore.Components.WebView.Wpf.BlazorWebView.StartWebViewCoreIfPossible()
   at Microsoft.AspNetCore.Components.WebView.Wpf.BlazorWebView.OnApplyTemplate()
   at System.Windows.FrameworkElement.ApplyTemplate()
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at System.Windows.Controls.Grid.MeasureOverride(Size constraint)
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at MS.Internal.Helper.MeasureElementWithSingleChild(UIElement element, Size constraint)
   at System.Windows.Controls.ContentPresenter.MeasureOverride(Size constraint)
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at System.Windows.Controls.Decorator.MeasureOverride(Size constraint)
   at System.Windows.Documents.AdornerDecorator.MeasureOverride(Size constraint)
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at System.Windows.Controls.Border.Meas
```
</details>

**Comment by SimplyAName** [comment]
Ratscanner is in the default Windows downloads folder

**Comment by Blightbuster** [comment]
Are you starting RatScanner through your start menu / some other programm / a shortcut?

**Comment by SimplyAName** [comment]
Launching it through the start menu it gave the error.
Launched from the folder it's in and worked fine so think you got it nailed

**Comment by Blightbuster** [comment]
Fixed in [v3.1.0](https://github.com/RatScanner/RatScanner/releases/tag/v3.1.0).
Please comment if this issue persists and I will reopen it.
